### ISSUE TITLE
TypeError on Python 3.7 

### ISSUE THREAD

**Issue by BytEvil** [celogeek/OctoPrint-ProgressBasedOnTime#3]
Hi.

```
File "/home/pi/OctoPrint/venv/lib/python3.7/site-packages/octoprint_ProgressBasedOnTime/__init__.py", line 24, in callback
    if result['printTimeLeft'] < 0:
TypeError: '<' not supported between instances of 'NoneType' and 'int'
```

Please fix this.

**Comment by celogeek** [comment]
Can you give me the list of plugins you got?
Maybe the log of octoprint to have more context?

**Comment by BytEvil** [comment]
Аfter removing your plugin Octocprint not crashing anymore. Here is [octoprint.log](https://pastebin.com/80rs0pDP)

**Comment by celogeek** [comment]
Fixed in 0.1.6

can you try to update?

**Comment by BytEvil** [comment]
After pathing worked as expected! Thanks for help. ;)
